Readme.md

```diff
@@ -158,4 +161,4 @@ That whisper.cpp project has an example which [uses](https://github.com/ggergano
 the same GGML implementation to run another OpenAI’s model, [GPT-2](https://en.wikipedia.org/wiki/GPT-2).<br/>
 It shouldn’t be hard to support that ML model with the compute shaders and relevant infrastructure already implemented in this project.
 
-If you find this useful, I’ll be very grateful if you consider a donation to [“Come Back Alive” foundation](https://savelife.in.ua/en/).
\ No newline at end of file
+If you find this useful, I’ll be very grateful if you consider a donation to [“Come Back Alive” foundation](https://savelife.in.ua/en/).
```

Comment:
what's the change here exactly? cr / crlf at the end?

Reply:
I don't intent to change this, but I don't know how to get rid of it. I'm sorry this snag jeopardizes the whoe PR.

Reply:
This PR allows edits by maintainers. I appreciate it If anyone helps me get rid of the snag